spec/adapters/jira-spec.js

```diff
@@ -10,6 +10,11 @@ const TICKETPAGE = `
       <a class="issue-link" data-issue-key="UXPL-39" id="key-val">UXPL-39</a>
       <h1 id="summary-val">A Random JIRA Issue</h1>
     </div>
+    <div class="issue-body-content">
+      <span id="type-val" class="value editable-field inactive">
+        <img alt="" src="/" title="Bug - A problem"> Bug
+      </span>
+    </div>
```

Comment:
How about adding a separate test case and HTML snippet (`BUGPAGE`) for this, so the tests also make sure the JIRA adapter distinguishes `bug` from `feature`? Like in the GitHub adapter for instance:

- https://github.com/bitcrowd/tickety-tick/blob/master/spec/adapters/github-spec.js#L14-L24
- https://github.com/bitcrowd/tickety-tick/blob/master/spec/adapters/github-spec.js#L69-L75